framework/src/modules/network/index.js

```diff
@@ -0,0 +1,63 @@
+const defaults = require('./defaults');
+const Network = require('./network');
+const BaseModule = require('../base_module');
+
+/* eslint-disable class-methods-use-this */
+
+/**
+ * Network module specification
+ *
+ * @namespace Framework.Modules
+ * @type {module.NetworkModule}
+ */
+module.exports = class NetworkModule extends BaseModule {
+	/* eslint-disable-next-line no-useless-constructor */
+	constructor(options) {
+		super(options);
+	}
+
+	static get alias() {
+		return 'network';
+	}
+
+	static get info() {
+		return {
+			author: 'LiskHQ',
+			version: '0.1.0',
+			name: 'lisk-core-network',
+		};
+	}
+
+	get defaults() {
+		return defaults;
+	}
+
+	get events() {
+		return ['ready'];
```

Comment:
Why not defining the events in the constructor?

Reply:
The controller gets the list of events which the module supports using `moduleInstance.events`. You cannot return from the constructor so events could not be exposed to the controller that way.